### ISSUE TITLE
Launch event archives not accessible from inside mybinder.org

### ISSUE THREAD

**Issue by betatim** [jupyterhub/mybinder.org-deploy#838]
The launch activity archives aren't accessible from a (notebook) pod running on mybinder.org:

```
$ wget https://archive.analytics.mybinder.org/events-2018-11-03.jsonl
--2018-11-10 20:51:32-- 
https://archive.analytics.mybinder.org/events-2018-11-03.jsonl
Resolving archive.analytics.mybinder.org (archive.analytics.mybinder.org)... 35.202.202.188
Connecting to archive.analytics.mybinder.org (archive.analytics.mybinder.org)|35.202.202.188|:443...
```

will eventually time out but works as expected from your laptop/computer outside the cluster.

https://mybinder.org is also not accessible so I think this is a network policy/firewall configuration issue. I think we should keep mybinder.org inaccessible from pods as it makes it harder to use a binder to launch binders.

ref #789

**Comment by minrk** [comment]
I think you're right. User pods don't have access to the cluster from the inside, and I'm going to guess that somehow this is blocking the request despite it being a public ip. I'm going to look at whitelisting access to https on the ingress controller to see if that works.

**Comment by minrk** [comment]
Should be fixed by #842

**Comment by minrk** [comment]
it worked!
